website/src/components/webinars/WebinarLayout.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { ReactNode, FunctionComponent } from 'react'
+
+import classNames from 'classnames'
+import styles from './WebinarLayout.module.scss'
+
+import { ContentSection } from '../content/ContentSection'
+import { breakpoints } from '../../breakpoints'
+import { useWindowWidth } from '../../hooks'
+import { useHubSpot } from '../../hooks/hubSpot'
+
+interface Customer {
+    name: string
+    logo: string
+    href: string
+}
+
+interface Speaker {
+    name: string
+    title: string
+    img: string
+    bio: string
+}
+
+interface Form {
+    formId: string
+    onFormSubmitted?: () => void
+}
+
+interface Props {
+    customer?: Customer
+    title: string
+    subtitle: string
+    description: ReactNode
+    form: Form
+    speakers: Speaker[]
+    children?: ReactNode
+}
+
+export const WebinarLayout: FunctionComponent<Props> = ({
+    title,
+    subtitle,
+    customer,
+    description,
+    form,
+    speakers,
+    children,
+}) => {
+    const windowWidth = useWindowWidth()
+    const isMdScreen = windowWidth < breakpoints.lg && windowWidth > breakpoints.sm
+    const isMobile = windowWidth < breakpoints.md
+
+    useHubSpot({
+        portalId: '2762526',
+        formId: form.formId,
+        targetId: 'form',
+        formInstanceId: form.formId,
```

Comment:
I'm curious what `formInstanceId` addition brings, can you explain @bretthayes ?

Reply:
Check out [the API](https://legacydocs.hubspot.com/docs/methods/forms/advanced_form_options#:~:text=in%20email%20inputs.-,formInstanceId,-O%2C%20I) for customizing the embed code. It was originally meant as a safeguard against any conflicts with multiple forms being used on the same page.

Reply:
Thanks, I wasn't sure since our other hubspot forms aren't using it on this repo and the form isn't used twice on this page. Should we add this prop to the other hubspot hooks?

Reply:
We can probably mark it as optional since we're not using dupe forms on these pages and in this template. But it may be nice to have for the future in case we go back down this route again.